Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/README.md

```diff
@@ -37,6 +37,18 @@ This integration was integrated and tested with version 1.0 of Microsoft Graph G
     Click&nbsp;<strong>Test</strong>&nbsp;to validate the new instance.
   </li>
 </ol>
+
+
+<h2>Use a self deployed Azure application</h2>
+<p>To use a self configured Azure application, a new Azure App Registration should be added in the Azure Portal. To add such registration, refer to the following Microsoft article:
+<a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app">https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app</a></p>
+<p>The Tenant ID, Client ID and Client secret are needed for the integration.
+To configure the integration in Demisto to use the application, place those parameters in the following manner (instead of you received from the admin consent in the current doc):
+<p>ID - Client ID<br>
+Token - Tenant ID<br>
+Key - Client Secret</p></p>
+
```

Comment:
we are missing the required permissions to set for the app

Reply:
It's written for each command separately and at the beginning in section `Detailed Description`
Would you like me to add it to the above section as well?

Reply:
yes please